docs/distributing-policies/publish-policy-to-artifact-hub.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: "Publish to Artifact Hub"
+title: ""
+---
+
+# Publish Policy to Artifact Hub
+
+[Artifact Hub](https://artifacthub.io/) is a website where users can find,
+install and publish packages and configurations for CNCF projects.
+
+Kubewarden policies can be published on Artifact Hub and made
+discoverable to the wide audience of CNCF users.
+
+:::note
+Artifact Hub is content aggregator, it doesn't actually host the
+artifacts that are published on it.
+
+Kubewarden Policies need to be physically hosted by a container
+registry or a web server.
+See the *"[distributing policies](../distributing-policies)"* section for more information about how to
+host your policies.
+:::
+
+## Prepare your Git repository
+
+Artifact Hub crawls Git repositories looking for special metadata files.
+
+There are different kind of layouts the Git repository can have. They
+are all documented in depth by the [official Artifact Hub documentation](https://artifacthub.io/docs/topics/repositories/#kubewarden-policies-repositories).
+
+Artifact Hub is pretty flexible, you can organize your code in these ways:
+
+* Have a Git repository dedicated to Artifact Hub: this repository will not contain
+  any policy source code. It will be a collection of the YAML files required by
+  Artifact Hub.
+* Add a Artifact Hub directory inside of the Git repository that holds the source
+  of your policy. This is an iteration of the previous approach, the only difference
+  is that it focuses just on one policy: the one defined inside of the Git repository.
+  This approach allows to keep multiple versions of the policy published on Artifact Hub.
+* Add the `artifacthub-pkg.yml` and the `artifacthub-repo.yml` files to the root
+  of the Git repository that holds the policy source code. This approach is the 
+  simplest one. The only limitation is that only the latest version of the policy
+  will be visible on Artifact Hub.
+
+The last approach is what is used inside of our official policy templates.
+The Git repository that is scaffolded will include the `artifacthub-pkg.yml`
+and `artifacthub-repo.yml` files.
+
+## Publishing Steps
+
+Before publishing a policy to Artifact Hub, you must create an account on
+the [website](https://artifacthub.io/). To do that, use the *"Sign Up"* button
```

Comment:
Just mentioning this should suffice. We do not want to explicitly provide click-this-button type steps which should be common knowledge. However, providing the context that signup is mandatory is relevant.
```suggestion
the [website](https://artifacthub.io/).
```